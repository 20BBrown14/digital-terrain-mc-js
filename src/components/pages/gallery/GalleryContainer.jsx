import React from 'react';
import GalleryView from './GalleryView';

/**
 * Gallery page container.
 */
const GalleryContainer = () => (
  <GalleryView />
);

export default GalleryContainer;
