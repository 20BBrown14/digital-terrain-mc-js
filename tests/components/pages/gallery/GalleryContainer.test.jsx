import React from 'react';
import GalleryContainer from '../../../../src/components/pages/gallery/GalleryContainer';

describe('GalleryContainer', () => {
  it('renders a default view', () => {
    const container = render(<GalleryContainer />);
    expect(container).toMatchSnapshot();
  });

  it('has a GalleryView', () => {
    const container = shallow(<GalleryContainer />);
    expect(container.find('GalleryView')).toHaveLength(1);
  });
});
