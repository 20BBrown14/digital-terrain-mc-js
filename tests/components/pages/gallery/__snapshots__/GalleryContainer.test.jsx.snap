// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryContainer renders a default view 1`] = `
<div>
  Gallery Page
</div>
`;
